src/registrar/templates/admin/base_site.html

```diff
@@ -18,11 +18,12 @@
   <link rel="apple-touch-icon" size="180x180" 
   href="{% static 'img/registrar/favicons/favicon-180.png' %}"
   >
+  <script src="{% static 'js/uswds-init.min.js' %}" defer></script>
+  <script src="{% static 'js/uswds.min.js' %}" defer></script>
```

Comment:
These two scripts are required for things like modals to work in django admin (as well as other uswds elements). 
This ticket initially used those for confirmation, but that was ultimately scrapped given some other complexities with it.

I'd suggest keeping this in because its likely that as admin expands, we will need to use other uswds components. That said though this is totally optional and can be removed, whatever is prudent.